src/Perspex.SceneGraph/RelativeRect.cs

```diff
@@ -191,14 +191,14 @@ public static RelativeRect Parse(string s, CultureInfo culture)
                     parts[2] = parts[2].TrimEnd('%');
                     parts[3] = parts[3].TrimEnd('%');
                     unit = RelativeUnit.Relative;
-                    scale = 0.01;
+                    scale = 0.01m;
                 }
 
                 return new RelativeRect(
-                    double.Parse(parts[0], culture) * scale,
-                    double.Parse(parts[1], culture) * scale,
-                    double.Parse(parts[2], culture) * scale,
-                    double.Parse(parts[3], culture) * scale,
+                    (double)(decimal.Parse(parts[0], culture) * scale),
```

Comment:
Ok removed decimal. I will create custom RelativeRect comparer, something like this: https://github.com/Perspex/Perspex/blob/01562bd98c20fec2608fcdc70c5e0cbed5a81c8b/tests/Perspex.Direct2D1.UnitTests/RectComparer.cs
https://github.com/Perspex/Perspex/blob/01562bd98c20fec2608fcdc70c5e0cbed5a81c8b/tests/Perspex.Direct2D1.UnitTests/RectComparer.cs